dm/pkg/log/string.go

```diff
@@ -11,11 +11,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package utils
```

Comment:
why move this file, static func which doesn't depends on any other package better stay in pkg/utils

Reply:
I remember other files in `utils` will cause a cycle import. Will try to revert it later.